assets/frontend/app/es6/login/login.component.html

```diff
@@ -1,7 +1,7 @@
 <section class="view__modal" ng-if="loginController.showLoginForm">
   <div class="modal__container login">
     <div class="login__header">
-      <span class="icon-nova"></span>
+      <span class="icon-exess"></span>
```

Comment:
this will not work :)

Reply:
I couldn't see anything not working

what should be done to make it work?